.github/workflows/checkstyle.yml

```diff
@@ -0,0 +1,20 @@
+name: CheckStyle
```

Comment:
All of the content of this workflow could be placed as an additional step in the already existent `CI.yml` workflow. We don't have separate workflows for our other projects and include `test`, `lint`, `build` all in a single workflow. Let's do that here too.

Reply:
Fixed in 5510fba